app/views/prior_authority/applications/show.html.erb

```diff
@@ -8,13 +8,33 @@
     <%= render 'prior_authority/shared/application_nav', application_summary: @summary, current_page: 'overview' %>
   </div>
   <div class="govuk-grid-column-two-thirds">
-
     <h2 class="govuk-heading-m"><%= t('.overview') %></h2>
-    <%= govuk_summary_card(title: t('.application_details')) { _1.with_summary_list(rows: @details.overview) } %>
+    <%= render 'overview_card', model: @details.overview_card %>
 
     <h2 class="govuk-heading-m"><%= t('.about_the_request') %></h2>
-    <%= govuk_summary_card(title: t('.primary_quote')) do |card| %>
-      <%= govuk_summary_list(rows: @details.primary_quote_rows) %>
+    <%= render 'primary_quote_card', model: @details.primary_quote_card %>
+    <%= render 'reason_why_card', model: @details.reason_why_card %>
+    <% @details.alternative_quotes.each_with_index do |alternative_quote, index| %>
+      <%= render 'alternative_quote_card', model: @details.alternative_quote_card(alternative_quote), n: index + 1 %>
```

Comment:
not 100% sure I see the advantage of this pattern over just having teh `alternative_quotes` method return the class??